.github/config/android-arm.json

```diff
@@ -8,7 +8,8 @@
       "2",
       "3",
       "black_box_test"
-    ]
+    ],
+    "test_attempts": 3
```

Comment:
I'd actually be a bit conservative and go with 2 out of the gate as a default. Device related errors haven't been too frequent recently, and I'm a bit cautious about just adding extra feedback delay if something is _really broken_.

If the normal test run gives you "its broken" signal in 15 minutes, now you have to wait 45 minutes.